sklearn/utils/__init__.py

```diff
@@ -471,9 +471,3 @@ def tosequence(x):
         return list(x)
```

Comment:
shouldn't these be imported for backwards compatibility? Or not because this is utils?